values-es/strings.xml

```diff
@@ -1330,44 +1330,44 @@
     <string name="draft_feature_slime00_title">Desastre de limo</string>
     <string name="draft_feature_slime00_text">Los desastres de limo se comerán tu ciudad. Por supuesto, sólo si quieres que lo haga.</string>
 
-    <string name="settings_terrainshading">Terrain shading</string>
-    <string name="draft_deco_chr_tree00_title">Christmas tree</string>
-    <string name="draft_deco_chr_tree00_text">The Christmas tree will fill the hearts of your inhabitants with joy and glory. Do you see any presents?</string>
-    <string name="draft_jardinehouse00_title">Jardine House</string>
-    <string name="draft_jardinehouse00_text">The Jardine House is a office tower placed in Hong Kong. It has 52 floors and has a height of 179 m. It was completed in 1972.</string>
-    <string name="draft_bankofchina00_title">Bank of China</string>
-    <string name="draft_bankofchina00_text">The Bank of China Tower in Hong Kong houses the headquarters for the Bank of China HK. It has 70 floors, is 367 m high and was finished in 1990.</string>
-    <string name="settings_road_zone_width">Width between zone roads</string>
-    <string name="settings_road_zone_height">Height between zone roads</string>
-    <string name="draft_feature_christmas_pack00_title">Christmas special</string>
-    <string name="draft_feature_christmas_pack00_text">The limited offer contains:</string>
-    <string name="draft_feature_christmas00_title">Christmas Time!</string>
-    <string name="draft_feature_christmas00_text">Various decorations for Christmas time :)</string>
-    <string name="draft_category_christmas00_title">Winter time</string>
-    <string name="draft_deco_chr_snowman00_title">Snowman</string>
-    <string name="draft_deco_chr_snowman00_text">Children like to build snowmen in winter time.</string>
-    <string name="draft_internationalcommercecentre00_title">ICC</string>
-    <string name="draft_internationalcommercecentre00_text">The International Commerce Centre in Hong Kong is a 108 storey, commercial skyscraper. It\'s the tallest building in Hong Kong with a height of 484 m. It was completed in 2010.</string>
-    <string name="settings_autosave_interval">Autosave interval</string>
-    <string name="settings_autosave_interval_off">Off</string>
+    <string name="settings_terrainshading">Sombreado del terreno</string>
+    <string name="draft_deco_chr_tree00_title">Árbol de Navidad</string>
+    <string name="draft_deco_chr_tree00_text">El árbol de Navidad llenará el corazón de los habitantes con gozo y gloria. ¿Ves algún regalo?</string>
+    <string name="draft_jardinehouse00_title">Casa Jardine</string>
+    <string name="draft_jardinehouse00_text">Jardine House es una torre de oficinas situada en Hong Kong. Tiene 52 pisos y tiene una altura de 179 m. Se completó en 1972.</string>
```

Comment:
The following string is poorly worded:"Draft_jardinehouse00_text" Jardine House has a height of 168 m.